msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,117 @@
+
+E.g. Graph permissions are not always about Teams data.
+For example, Your AAD app may need Mail.Read permission, but this is not an RSC scenario because it is not about team data, and team owners cannot give consent to other's mailboxes. APIs for Files, SharePoint, OneNote, Planner, and Calendar within a team do not support RSC yet.
+
+#### 3. Update your Teams app manifest to link to your AAD app ID
+Add a [webApplicationInfo](/microsoftteams/platform/resources/schema/manifest-schema#webapplicationinfo) section to the manifest that includes two properties - `id` is the AAD app ID, and `applicationPermissions` are the RSC permissions (scopes) your app requires.
+
+> **Note**: If your app needs non-RSC permissions, you can continue to use them as always, but they do not go in the manifest. 
+
+```
+{ 
+  "$schema": ".../MicrosoftTeams.schema.json", 
+   "webApplicationInfo": { 
+      "id": "cb38cf54-ac89-4a7a-9ea3-095d3d080037", 
+      "applicationPermissions": [  
+      "File.Read.Group",
+      "TeamSettings.Read.Group",
+      "ChannelMessage.Read.Group",
+      "Member.Read.Group"
+      ] 
+  }
+  ... 
+} 
+```
+<!Refer to the list of available Graph permissions for RSC in [Teams permissions](../graph/permissions-reference?context=graph%2Fapi%2Fbeta&view=graph-rest-beta.md#Teams permissions)section.>
+Refer to the list of available Graph permissions for RSC in <a href="https://docs.microsoft.com/en-us/graph/permissions-reference?context=graph%2Fapi%2Fbeta&view=graph-rest-beta">Teams permissions</a> section.
+When you make an RSC API call, Graph doesn't know on behalf of which user (if any) you are doing the work. Hence, you should be aware of the use cases such as creating a channel or removing a tab, as you are calling Application permissions rather than delegated permissions.
+
+#### 4. Get an access token 
+
+Before you make a REST call to the Graph, you need to [get an access token](/graph/auth-v2-service) for the application permissions similar to getting an application permission token for non-RSC use.
+You specify the pre-configured permissions by passing `https://graph.microsoft.com/.default` as the value for the `scope` parameter in the token request. See the `scope` parameter description in the token request below for details.
+
+### Token request
+
+```
+POST https://login.microsoftonline.com/{tenant}/oauth2/v2.0/token
+Host: login.microsoftonline.com
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=client_credentials
+&client_id={appId}
+&client_secret={appSecret}
+&scope=https%3A%2F%2Fgraph.microsoft.com%2F.default
+
+```
+
+| Parameter     | Condition | Description 
```

Comment:
Can we shorten this section up? Let's get rid of the tables, we already have docs we can link to for that content.

Reply:
Done